book/chrome.md

```diff
@@ -656,41 +681,49 @@ Each tab needs to make sure not to draw to those pixels:
 class Tab:
     def draw(self, canvas):
         for cmd in self.display_list:
-            if cmd.top > self.scroll + HEIGHT - CHROME_PX: continue
+            if cmd.top > self.scroll + HEIGHT - self.chrome_bottom:
+                continue
             if cmd.bottom < self.scroll: continue
-            cmd.execute(self.scroll - CHROME_PX, canvas)
+            cmd.execute(self.scroll - self.chrome_bottom, canvas)
 ```
 
-There are still sometimes going to be halves of letters that stick out
+Now let's turn our attention to designing the UI.
+
+First, there are still sometimes going to be halves of letters that stick out
 into the browser chrome, but we can hide them by just drawing over
-them:
+them (here I'm assuming we've already computed `self.bottom` (representing
+the bottom y coordinate of the browser chrome); that will
+come in a moment):
 
 ``` {.python}
-class Browser:
-    def paint_chrome(self):
+class Chrome:
+    def paint(self):
         cmds = []
-        cmds.append(DrawRect(0, 0, WIDTH, CHROME_PX, "white"))
+        cmds.append(
+            DrawRect(0, 0, WIDTH, self.bottom, "white"))
         return cmds
 ```
 
 You'll also need to adjust `scrolldown` to account for the height of
-the page content now being `HEIGHT - CHROME_PX`:
+the page content now being `HEIGHT - self.bottom`:
```

Comment:
Hmm, we've been passing `chrome_bottom` to each `Tab`, but in Chapter 15 we instead seem to care about `frame_height`. Should we pass `HEIGHT - self.bottom` instead? It will align more with Chapter 15 and also the code will be a tiny bit nicer.